### ISSUE TITLE
Issue installing Algokit on MacOS

### ISSUE THREAD

**Issue by mvieiradcu** [algorandfoundation/algokit-cli#471]
###  algokit fail to install

I am trying to install algokit on Intel MacOS (14.2.1 (23C71)) but it shows an error `Unhandled ModuleNotFoundError: No module named 'yamlinclude'` 

I tried install using pipx and brew both the same error

` pipx install algokit` 
### Your environment

algokit doctor
Unhandled ModuleNotFoundError: No module named 'yamlinclude'

### Steps to reproduce

1. install algokit 
2. try to run any command like `algokit --version`
<img width="452" alt="image" src="https://github.com/algorandfoundation/algokit-cli/assets/127679938/85075209-c8e3-40c0-a763-475e63474950">

**Comment by AiCEG** [comment]
I second the issue.
Trying to install the yamlinclude module just results in an error (no matching distribution found, from versions: none)

So far I figured that the error comes from the copier modules template.py.
As a quickfix I commented the lines using the yamlinclude.

**Comment by aorumbayev** [comment]
Hello @AiCEG @mvieiradcu thanks for reporting.

Can you share the version of python you are using?

**Comment by AiCEG** [comment]
@aorumbayev I am using version: 3.12

**Comment by mvieiradcu** [comment]
I am using Python 3.12.2

**Comment by aorumbayev** [comment]
we will have a proper fix some time tomorrow,

in the meantime can you try a temporary hotfix by installing from the fix branch and let me know if this fixes it for you?

```sh
pipx install git+https://github.com/algorandfoundation/algokit-cli@fix/yaml-transient-dep --force
```

**Comment by aorumbayev** [comment]
@AiCEG @mvieiradcu

**Comment by FernandoMIMO** [comment]
![AlgoKitYamlIncludeIssue](https://github.com/algorandfoundation/algokit-cli/assets/146851124/8f781584-5702-41dc-8fa6-7c3eb1292d09)


Got this issue today, see you guys working on exact issue, and the fix worked for me. I just ran the line that @aorumbayev shared an hour ago in the chat above and got the result in the picture.

**Comment by neilcampbell** [comment]
This appears to be the issue https://github.com/copier-org/copier/issues/1568
We'll monitor the resolution on the `copier` side, however in the meantime we'll pin to `pyyaml-include = "1.4.1"`, which fixes the issue.

**Comment by neilcampbell** [comment]
@AiCEG @mvieiradcu This is fixed in `2.0.2`, which is being published right now.

**Comment by AiCEG** [comment]
Just tested it aswell. Works! Thank you guys
